program-runtime/src/pre_account.rs

```diff
@@ -182,8 +182,11 @@ impl PreAccount {
         static ZEROS: [u8; ZEROS_LEN] = [0; ZEROS_LEN];
```

Comment:
This array could be turned into a slice here. (just add `&` to the type and value)
Then https://doc.rust-lang.org/std/primitive.slice.html#method.get could be used below.